File: 3300-minimum-element-after-replacement-with-digit-sum/3300-minimum-element-after-replacement-with-digit-sum.py
class Solution(object):
    def minElement(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        for i in range(len(nums)):
            sum=0
            while(nums[i]!=0):
                sum+=nums[i]%10
                nums[i]//=10
            nums[i]=sum
        return min(nums)

File: 3300-minimum-element-after-replacement-with-digit-sum/3300-minimum-element-after-replacement-with-digit-sum.cpp
class Solution {
public:
    int minElement(vector<int>& nums) {
        for(int i=0; i<nums.size(); i++)
        {   
            int a = nums[i];
            int sum = 0;
            while (a!=0)
            {
                sum = sum + (a % 10);
                a = a / 10;
            }
            nums[i]=sum;
        }
        int min=nums[0];
        for (int i=0; i<nums.size(); i++)
        {
            if (nums[i]<min) min=nums[i];
        }
        return min;
    }
};